app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -269,11 +269,11 @@ BrowserWorld::State::EnsureControllerFocused() {
       break;
     }
 
-    if (!controller.leftHanded) {
+    if (!controller.leftHanded && controller.enabled) {
       right = &controller;
     }
 
-    if (!first) {
+    if (!first && controller.enabled) {
```

Comment:
I am a little confused with this `first`'s meaning. Why don't we replace it with `left`?

Reply:
Because it is the first enabled controller discovered, which might be the right hand controller. It can probably be cleaned up. Originally, I just gave the first controller discovered focus, but on the Quest it seems the order is random. So I added the code to default to the right hand until I found a way to default to the controller than has focus in the home environment.